README.md

```diff
@@ -67,25 +64,25 @@ exceptions.
 
 If we would like to ignore `ExcessRaiseError`s (because we know that
 an underlying function will raise an exception), then we would add its
-error code to a file named *.darglint*:
+error code to a file named *.darglint2*:
 
 ```ini
-[darglint]
+[darglint2]
 ignore=DAR402
 ```
```

Comment:
@9dogs, do you agree that it's good to rename the configuration section to `[darglint2]` for consistency, too?

Reply:
100%